csharp-basic-topics/MemoryStreamInCsharp/MemoryStreamInCsharp/Program.cs

```diff
@@ -0,0 +1,145 @@
+﻿using MemoryStreamInCsharp.Properties;
+using System.Text;
+using MemoryStreamInCsharp;
+
+var phrase1 = "How to Use MemoryStream in C#";
+var phrase1Bytes = Encoding.UTF8.GetBytes(phrase1);
+var phrase2 = " - explanation with examples";
+var phrase2Bytes = Encoding.UTF8.GetBytes(phrase2);
+var memoryStream = new MemoryStream();
+
+SimpleConstructor();
+ByteArrayConstructor();
+FullConstructor();
+
+WriteToMemoryStream();
+ReadFromMemoryStream();
+
+LoadImageFromResources();
+
+SerializeAndDeserializeObject();
+
+string ShowMemoryStreamProperties(MemoryStream memoryStream, string comment = "")
+{
+    var sb = new StringBuilder();
+    if (!string.IsNullOrEmpty(comment))
+        sb.AppendLine($"{comment}\n--------------------------");
+
+    sb.AppendLine($"{"Length:",-20}{memoryStream.Length}");
+    sb.AppendLine($"{"Capacity:",-20}{memoryStream.Capacity}");
+    sb.AppendLine($"{"CanRead:",-20}{memoryStream.CanRead}");
+    sb.AppendLine($"{"CanSeek:",-20}{memoryStream.CanSeek}");
+    sb.AppendLine($"{"CanWrite:",-20}{memoryStream.CanWrite}");
+    sb.AppendLine($"{"CanTimeout:",-20}{memoryStream.CanTimeout}");
+    sb.AppendLine($"{"publiclyVisible:",-20}{memoryStream.TryGetBuffer(out _)}");
+
+    return sb.ToString();
+}
+
+void SimpleConstructor()
+{
+    var memoryStream = new MemoryStream();
+    var displyProperties = ShowMemoryStreamProperties(memoryStream, 
```

Comment:
On a multiple places throughout the code, you have name `displyProperties`, but instead, it should be `displayProperties`. I didn't notice this on my first review, so can you please fix it?